packages/browser-wallet/src/popup/pages/Setup/ChooseNetwork.tsx

```diff
@@ -3,15 +3,28 @@ import React from 'react';
 import { useNavigate } from 'react-router-dom';
 import Button from '@popup/shared/Button';
 import { absoluteRoutes } from '@popup/constants/routes';
-import { useSetAtom } from 'jotai';
+import { useSetAtom, useAtomValue } from 'jotai';
 import { useTranslation } from 'react-i18next';
+import { seedPhraseAtom } from '@popup/state';
 import { networkConfigurationAtom } from '@popup/store/settings';
-import { mainnet, testnet } from '../NetworkSettings/NetworkSettings';
+import { setupRoutes } from './routes';
+// TODO Remove stagenet
+import { mainnet, testnet, stagenet } from '../NetworkSettings/NetworkSettings';
 
 export function ChooseNetwork() {
```

Comment:
I feel like the texts on this page should be directed towards recovery when that is the context, perhaps @jens-concordium can chip in.

Reply:
I split up the description based on whether we are in recovery or not.
(Not sure about the actual text though)